experiments/ClimaEarth/run_amip.jl

```diff
@@ -166,8 +163,31 @@ land_sim_name = "bucket"
 t_end = Float64(time_to_seconds(config_dict["t_end"]))
 t_start = 0.0
 tspan = (t_start, t_end)
-Δt_component = Float64(time_to_seconds(config_dict["dt"]))
 Δt_cpl = Float64(config_dict["dt_cpl"])
+component_dt_names = [:dt_atmos, :dt_land, :dt_ocean, :dt_seaice]
+# if any of the component dt's are not specified, set them all to the overall dt
```

Comment:
Could you reword this comment to emphasize that the `if` branch uses individual component model dts, and the `else` branch ignores them to use the general dt?